InstrumentDB/1 Table.sql

```diff
@@ -0,0 +1,54 @@
+use InstrumentDB
+go
+
+drop table if exists dbo.Instrument
+go
+
+create table dbo.Instrument(
+     InsturmentID int not null identity primary KEY,
+     CustomerName varchar(40) not null constraint ck_CustomerName_cannot_be_blank check(CustomerName<>''),
+     InstrumentName varchar(30) not null constraint ck_InsturmentName_cannot_be_blank check(InstrumentName<>''),
+     Category varchar(20) not null constraint ck_Category_cannot_be_blank check(Category<>''),
+     MonthlyRentalFee decimal(5,2) not null,
+     DateRented date not null,
+     DateReturned date,
+     TotalProfit as datediff(day, DateRented, DateReturned)/30*MonthlyRentalFee,
```

Comment:
-- LL for total profit would be better to use the month as your interval in your datediff. Not all months have 30 days...